humminbird.h

```diff
@@ -48,7 +49,7 @@ class HumminbirdBase
 
   /* Constants */
 
-  static constexpr const char* humminbird_icons[] = {
+  QStringList humminbird_icons = {
```

Comment:
I would recommend this be a const QStringList.  That avoids any checks for detachment as well as alleviates the need to use qAsConst with this list, so you save the two uses you added in humminbird.cc as well.